### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by Zumbalamambo** [RobotEdh/Yolov-4#1]
How do I train with pascal voc?

**Comment by RobotEdh** [comment]
hello,

I need to build first a loss function for Tensorflow working for Yolo4
I just started so I will keep you posted

**Comment by RobotEdh** [comment]
Done, i have juste released the loss function forYolov4 and anexample of training based on the Pascal VOC dataset.

**Comment by RobotEdh** [comment]
issue closed
